fe1-web/__tests__/model/network/FromJson.ts

```diff
@@ -0,0 +1,496 @@
+/* eslint-disable */
+
+import {
+  ActionType, CloseRollCall,
+  CreateLao,
+  CreateMeeting, CreateRollCall,
+  ObjectType, OpenRollCall,
+  StateLao, StateMeeting,
+  UpdateLao, WitnessMessage
+} from '../../../Model/Network/Method/Message/data';
+import { ProtocolError } from '../../../Model/Network';
+import {Base64Data, Hash, PrivateKey, PublicKey, Signature, Timestamp} from "../../../Model/Objects";
+import { sign } from "tweetnacl";
+import { encodeBase64 } from "tweetnacl-util";
+
+const assert = require('assert');
+const assertChai = require('chai').assert;
```

Comment:
there's normally no reason to use chai if we are on Jest. Was there a specific need that required chai?

Reply:
There was before I deleted the mocha`package (as well as the typescript package it seems ^^') on the commit you reviewed. We could indeed use the expect syntax from jest.

I keep a note of this and will remove chai once the priority list becomes smaller